server/schedulers/balance_benchmark_test.go

```diff
@@ -83,6 +84,40 @@ func newBenchCluster(ctx context.Context, ruleEnable, labelEnable bool, tombston
 	return tc
 }
 
+func newBenchBigCluster(ctx context.Context, storeNumInOneRack, regionNum int) *mockcluster.Cluster {
+	opt := config.NewTestOptions()
+	tc := mockcluster.NewCluster(ctx, opt)
+	opt.GetScheduleConfig().TolerantSizeRatio = float64(storeCount)
+	opt.SetPlacementRuleEnabled(true)
+
+	config := opt.GetReplicationConfig()
+	config.LocationLabels = []string{"az", "rack", "host"}
+	config.IsolationLevel = "az"
+
+	storeID, regionID := uint64(0), uint64(0)
```

Comment:
Avoid stores with ID = 0

Reply:
I can see L109, And I wll change code in `newBenchCluster`